javaparser-core/src/main/java/com/github/javaparser/ast/validator/chunks/ModifierValidator.java

```diff
@@ -49,7 +49,7 @@ public void visit(ClassOrInterfaceDeclaration n, ProblemReporter reporter) {
 
     private void validateClassModifiers(ClassOrInterfaceDeclaration n, ProblemReporter reporter) {
         if (n.isTopLevelType()) {
-            validateModifiers(n, reporter, PUBLIC, ABSTRACT, FINAL, STRICTFP);
+            validateModifiers(n, reporter, PUBLIC, ABSTRACT, STATIC, FINAL, STRICTFP);
```

Comment:
Are you saying that java.jj does not handle nested/inner classes?  Or that your changes to java.jj break java.jj's handling of nested/inner classes?

We would like every Java file to be a legal stub file.  That goal is not possible if stub files prohibit nested/inner classes.